.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,12 +1,47 @@
 ---
 name: Feature Request
-about: You want to suggest a new feature for Atmosphère.
-
+about: Suggest a new feature for Atmosphère.
+#assignees:
 ---
 
 ## Feature Request
 
+[ If any section does not apply or you do not have the information needed for that section, use N/A (Not Applicable). ]  
+[ Lines between [ ] (square brackets) are to be removed before posting. ]
+
+[ Please refer to the already existing [feature requests](https://github.com/Atmosphere-NX/Atmosphere/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22features%2Ffeature-request%22) before you make your own request. ]  
```

Comment:
```suggestion
[ Please search for existing [feature requests](https://github.com/Atmosphere-NX/Atmosphere/issues?utf8=%E2%9C%93&q=is%3Aissue+label%3A%22features%2Ffeature-request%22) before you make your own request. ]
```
Remove the `is: open` filter. We don't want duplicates of closed feature requests.